src/configurations/HibernateUtils.java

```diff
@@ -0,0 +1,38 @@
+package configurations;
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.Metadata;
+import org.hibernate.boot.MetadataSources;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.service.ServiceRegistry;
+
+public class HibernateUtils {
+
+	private static final SessionFactory sessionFactory = buildSessionFactory();
+
+	private static SessionFactory buildSessionFactory() {
+		try {
+
+			ServiceRegistry serviceRegistry = new StandardServiceRegistryBuilder()
+					.configure("hibernate.cfg.xml").build();
+
+			Metadata metadata = new MetadataSources(serviceRegistry).getMetadataBuilder().build();
+
+			return metadata.getSessionFactoryBuilder().build();
+		}
+		catch (Throwable ex) {
+
+			System.err.println("Initial SessionFactory creation failed." + ex);
+			throw new ExceptionInInitializerError(ex);
```

Comment:
Throwing error makes no sense. Please add the throws to the signature then catch the error at the caller